index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/b2291adafa.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile version</title>
+</head>
+<body>
+        <section class="main-container">
+           <Section class="header">
+               <Nav>
+                   <a href=""><h3>My Logo</h3></a>
+                   <span class="material-icons">menu</span>
+               </Nav>
+           </Section>
+           <section class="headline">
+                   <h2>I'm Esther <br>Glad to see you!</h2>
+                   <p>I'm a software developer
+                      I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and a project you need coded, don't hesitate to contact me.
+                   </p>
+                   <h5>LET'S CONNECT</h5>
+           </section>
+           <section class="icons">
+            <ul>
+                <li><img src="/Imgs/Vector.svg" alt=""></li>
+                <li><img src="/Imgs/linkedin.svg" alt=""></li>
+                <li><img src="/Imgs/M-Vector.svg" alt=""></li>
+                <li><img src="/Imgs/github.svg" alt=""></li>
+                <li><img src="/Imgs/Victory.svg" alt=""></li>
```

Comment:
- [ ] Great job using `li` tags for listings! I would suggest adding `a` tags inside the `li` tags as the previous reviewer said.